app/main/views/search.py

```diff
@@ -23,6 +23,7 @@
 
 @main.route('/search/suppliers')
 def supplier_search():
+    sort_by = request.args.get('sort_by', 'asc')  # asc or desc
```

Comment:
We're only sorting by name now, but the designers have previously talked about adding other sort criteria.  You should make this "name_asc" and "name_desc" (or, better, just use "name" and add a separate parameter for the asc/desc so you don't have to mess with the strings here).